src/Util.cc

```diff
@@ -527,6 +527,86 @@ std::string validTopic(const std::vector<std::string> &_topics)
   }
   return std::string();
 }
+
+//////////////////////////////////////////////////
+Entity entityFromMsg(const EntityComponentManager &_ecm,
+    const msgs::Entity &_msg)
+{
+  if (_msg.id() != kNullEntity)
+  {
+    return _msg.id();
+  }
+
+  // If there's no ID, check name + type
+  if (_msg.type() == msgs::Entity::NONE)
+  {
+    return kNullEntity;
+  }
+
+  auto entities = entitiesFromScopedName(_msg.name(), _ecm);
+  if (entities.empty())
+  {
+    return kNullEntity;
+  }
+
+  for (const auto entity : entities)
```

Comment:
```suggestion
  for (const auto &entity : entities)
```
```suggestion
  for (const auto entity : entities)
```

Reply:
I think this isn't very important because `Entity` is just a POD. But I guess this could change in the future, so it doesn't hurt